library/errata_tool_product.py

```diff
@@ -138,12 +148,19 @@ def validate_params(module, params):
     for state in params['valid_bug_states']:
         if state not in BUGZILLA_STATES:
             raise InvalidInputError('valid_bug_states', state)
+
     solution = params['default_solution'].upper()
     try:
         common_errata_tool.DefaultSolutions[solution]
     except KeyError:
         raise InvalidInputError('default_solution', solution)
 
+    exd_org_group = params['exd_org_group']
+    try:
+        common_errata_tool.ExdOrgGroups[exd_org_group]
```

Comment:
After I coded this `InvalidInputError` class, I found out later that the Ansible API permits a `choices=` parameter for the argspec. We can use that instead.

Reply:
See `library/errata_tool_cdn_repo.py` for an example of the `choices=` parameter for the module argspec.

Reply:
Ack, nice.